### ISSUE TITLE
Interaction between ibus and Qt/QtCreator

### ISSUE THREAD

**Issue by fujiwarat** [ibus/ibus#1757]
```
What version of the product are you using? On what operating system?
OS (Linux distributions, UNIX or ...): Ubuntu 14.04
Architecture (i386, x86_64): x86_64
IBus version: 1.5.5-1ubuntu3
Input method name and version:
Python version: 2.7.5-5ubuntu3
dbus version: 1.6.18-0ubuntu4.1
dbus-python version: 1.2.0-2build2
qt version (if bug is about qt applications): 5.2.1
QtCreator version: 3.0.1-0ubuntu4

What steps will reproduce the problem?
What is the expected output? What do you see instead?

Please check:
https://bugreports.qt-project.org/browse/QTCREATORBUG-12371
where the problem is described.

In short there is a problem with interaction with QtCreator that goes away when I exit
ibus.  So there must be some problem with interaction between Qt (or QtCreator) and
ibus and I'd like ibus developers to look at it.

Quoting qtcreator plugin developer from the issue mentioned above: "Seems like the
application does not [get] keys delivered properly if ibus is running"

Thank you in advance
Best regards
Andrzej

PS. This is not a major issue - but an annoying one :)
```

Original issue reported on code.google.com by `Andrzej.Ostruszka` on 2014-09-03 11:06:36
